packages/hadron-style-manager/test/.compiled-less/da39a3ee5e6b4b0d3255bfef95601890afd80709/imports.json

```diff
@@ -0,0 +1 @@
+{"importedFiles":[".compiled-less/da39a3ee5e6b4b0d3255bfef95601890afd80709/imports.json","mocha.opts","style-manager.test.js","test.html.template","test.less"]}
```

Comment:
You probably don't want to commit this (and `packages/hadron-style-manager/test/test.html`)

Reply:
Hmm. Will fix.

Reply:
Probably a git merge stuffup.